app/index.html

```diff
@@ -125,10 +124,7 @@
             <span>{{ selectedSamplesNum }} samples / {{ selectedPatientsNum }} patients</span>
           </span>
         </div>
-        <save-virtual-study :show-save-button="showSaveButton" :stats="stats"
-                           :update-stats.sync="updateStats"></save-virtual-study>
-        <share-virtual-study :show-share-button="showShareButton" :stats="stats"
-                             :update-stats.sync="updateStats"></share-virtual-study>
+        <virtual-study v-if="showVirtualStudyButton" :stats="stats" :update-stats.sync="updateStats"></virtual-study>
```

Comment:
Doesn't we require two control parameters each for share and save button?. Or we can show share button for default and a controlled save button.

Reply:
After discussion, I will bring it back.